src/main/ui/app/component/NamedList.jsx

```diff
@@ -27,16 +27,13 @@ export default class NamedList extends Component {
         const COLUMNS_IF_SIZE = 5;
```

Comment:
Please also delete this, as well as the "half" function above. These are no longer used, right?